benchmark/aiperf/README.md

```
@@ -34,7 +34,7 @@ To use the provided configurations, you need to create accounts at <https://buil
 2. **Install dependencies in the virtual environment**
 
    ```bash
-   $ pip install aiperf huggingface_hub typer
+   $ pip install aiperf huggingface_hub typer httpx
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)